src/jupyter_contrib_nbextensions/nbextensions/load_tex_macros/main.js

```diff
@@ -0,0 +1,38 @@
+define(function(require, exports, module) {
+    var Jupyter = require('base/js/namespace');
+
+    function loadLatexUserDefs() { 
+        $.get('latexdefs.tex').done(function(data) {
+            data = data.replace(/^/gm, '\$\$\$').replace(/$/gm, '\$\$\$');
+            if ($('#latexdefs').length > 0) $('#latexdefs').remove();
+            $('body').append($('<div/>').attr('id', 'latexdefs').text(data));
+            console.log('latex_envs: loaded user LaTeX definitions latexdefs.tex');
+            onMarkdownCellRendering();
+        }).fail(function() {
+            console.log('load_tex_macros: failed to load user LaTeX definitions latexdefs.tex')
+        });
+    }
+
+    function rerenderMaths() { // probably something like that
+            MathJax.Hub.Queue(
+              ["PreProcess", MathJax.Hub], ["Reprocess", MathJax.Hub]
+            );
```

Comment:
I think 
```
MathJax.Hub.Queue(
  ["resetEquationNumbers",MathJax.InputJax.TeX],
  ["PreProcess",MathJax.Hub],
  ["Reprocess",MathJax.Hub]
);
``` 
could be better (resets equation numbers before reprocessing avoid an issue with labels in equations that cannot be rendered several times without reset.